crates/virtual-node/src/lib.rs

```diff
@@ -111,47 +111,67 @@ impl VirtualNode {
         VirtualNode::Text(text.into())
     }
 
-    /// Create a new `VirtualDomElement`.
+    /// Create a new [`VirtualNodeElement`] with the specified tag name.
     ///
-    /// This is the variant that is contained inside `VirtualNode::Element`. It
-    /// can be turned into a `VirtualNode` using the from/into.
+    /// This is the variant that is contained inside [`VirtualNode::Element`].
+    /// It can be turned into a [`VirtualNode`] using the from/into.
+    ///
+    /// [`VirtualNode`]: enum.VirtualNode.html
```

Comment:
ohhhhh that's how you do links in rustdoc.. thanks TIL!

Reply:
Actually it's standard markdown (commonmark flavor), you could also do this:

```
[`VirtualNode`](enum.VirtualNode.html]
```

But the separate links make the text easier to read in plain text :slightly_smiling_face:

Reply:
Oh sorry I meant the `enum.VirtualNode.html` for linking to an enum - thanks!